Iterable-iOS-SDK.podspec

```diff
@@ -76,7 +76,7 @@ Pod::Spec.new do |s|
 
   s.source_files = "swift-sdk/**/*.{h,m,swift}"
 
-  s.resources = ['swift-sdk/Resources/**/*.{storyboard,xib,xcassets}']
```

Comment:
We should create a separate PR for podspec changes. Also need to test thoroughly after making this change.